pino.js

```diff
@@ -75,6 +75,28 @@ Object.defineProperty(pinoPrototype, 'levelVal', {
 
 Object.defineProperty(pinoPrototype, '_setLevel', {
   value: function _setLevel (level) {
+    if (typeof level === 'object') {
+      // process and save rules for future offspring
+      // rules is an ordered (more specific first) list of [regexp, level]
+      this.levelRules = Object.keys(level).sort(tools.orderByPrecision).map((pattern) => {
```

Comment:
I like arrow functions too, but the code style of the project uses the full `function` declaration style.